alpharaw/match/psm_match_alphatims.py

```diff
@@ -231,7 +229,6 @@ def match_ms2_one_raw(
             np.inf if a fragment is not matched.
 
         """
-        self._preprocess_psms(psm_df_one_raw)
```

Comment:
intentionally removed?

Reply:
Yes, this function does nothing after I checked all sub-classes ...